techmatters/terraso-mobile-client
# feat: cache elevation data in client redux slice

## Description

Add a new slice to the terraso mobile client which handles elevation caching based on 5 decimal places of precision. The data hook will attempt to load elevation once (does not retry on failure).

### Checklist

### Related Issues
#1547

### Verification steps

Select the same location twice on the home screen (within 5 decimal places of precision). Verify that the elevation value is already loaded.
